### ISSUE TITLE
firebase-crashlytics-gradle:2.9.6 build fails with NegativeArraySizeException

### ISSUE THREAD

**Issue by tellypresence** [firebase/firebase-android-sdk#5096]
* Android Studio version: Flamingo patch 2
  * Firebase Component: firebase-crashlytics-gradle
  * Component version: 2.9.6

### Describe the problem
Building debug variant of android project using firebase-crashlytics-gradle:2.9.6 fails w/error:
```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:injectCrashlyticsBuildIdsDebug'.
Caused by: java.lang.NegativeArraySizeException: -5
```

**Comment by argzdev** [comment]
Hi @tellypresence, thanks for reporting. I can't reproduce the same behavior. Not sure what I'm missing. Can you share with me your build.gradle for both project and app module. It'll be very helpful to us if you have a minimal reproducible example. Please do remove any PII (Personal Identifiable Information) if needed. Thanks!

**Comment by sabby** [comment]
Same issue as [this one](https://github.com/firebase/firebase-android-sdk/issues/4720). I'm still seeing it in 2.9.6 also.

**Comment by argzdev** [comment]
Yeah, I think it is the same issue. Thanks for mentioning that, actually. For better tracking, kindly continue your report on that thread. Please do leave a thumbs up reaction emoji on the author's post so we can track the severity. I'll go ahead and close this one. Thanks!
